ssms/basic_simulators/drift_functions.py

```diff
@@ -132,3 +132,126 @@ def ds_conflict_drift(
     v_t = (w_t * coherence_t) + (w_d * coherence_d)
 
     return v_t  # , w_t, w_d
+
+def SSP_drift(
```

Comment:
I would align defaults and take `t = np.arange(0, 20, 0.01)` here for consistency. 

(Later this is overwritten when calling the `simulator` function anyways)